securedrop/journalist_app/__init__.py

```diff
@@ -17,10 +16,9 @@
 
 from db import db
 from journalist_app import account, admin, api, main, col
-from journalist_app.utils import (get_source, logged_in,
-                                  JournalistInterfaceSessionInterface,
-                                  cleanup_expired_revoked_tokens)
-from models import InstanceConfig, Journalist
+from journalist_app.sessions import Session, session
```

Comment:
That's mostly correct. I like the idea of importing the `Session` class as something like `SessionProvider`. I would like to keep the other one as `session` since it's supposed to be equivalent to `from flask import session`, so aside from the different import line, someone reading the code should be able to understand it as mostly equivalent to flask's built-in session stuff.